cmd/compute-domain-controller/main.go

```diff
@@ -57,6 +57,8 @@ type Flags struct {
 	httpEndpoint string
 	metricsPath  string
 	profilePath  string
+
+	previousDriverNamespaces cli.StringSlice
```

Comment:
Can we use the name `additionalNamespaces` instead? Intuitively, it reads better to name this `additionalNamespaces` and probably explain in the documentation why the DRA driver needs access to other namespaces

Reply:
Done